source/security/authentication.txt

```diff
@@ -0,0 +1,554 @@
+.. _kotlin-sync-auth:
+
+=========================
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: validate credentials, protocols, code example 
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using each
+**authentication mechanism** available in the MongoDB Community Edition.
+Authentication mechanisms are processes by which the driver and server
+confirm identity and establish trust to ensure security.
+
+To select a specific authentication mechanism, you can specify the
+mechanism, your credentials, and other necessary information
+in the options of your connection string or in a ``Credential`` struct.
+
+To learn more about the connection string options for authentication,
+see the :manual:`Authentication Options
```

Comment:
Link should be: https://www.mongodb.com/docs/manual/reference/connection-string-options/#authentication-options

Reply:
Linked to https://www.mongodb.com/docs/manual/reference/connection-string/#connection-string-formats, which leads to helpful information on string format options as well as link to the above page^

Reply:
I'll remove the link entirely. Initially wanted to include because I ran into trouble with ssl being enabled depending on whether the srv or standard string format was used. I've since updated the tutorial samples, so hopefully that won't be an issue for users any more.